src/AltinnCore/Common/Factories/ModelFactory/XsdToJsonSchema.cs

```diff
@@ -1634,6 +1710,18 @@ private bool IsTopLevel(XmlSchemaObject item)
             return item.Parent == mainXsd;
         }
 
+        private void LogInfo(string msg)
+        {
+            _logger.LogInformation(msg);
+            Trace.WriteLine("Info: " + msg);
```

Comment:
Whats the need for Trace.Writeline here?

Reply:
To actually get the log line displayed in unit test. If you can make ILogger log unit tests, please tell me how   :-)